utils/multisig.ts

```diff
@@ -84,9 +86,11 @@ export class Multisig {
     funcName: string,
     args: any[],
     signers: ethers.Wallet[]
-  ): Promise<ethers.types.Transaction> {
+  ): Promise<any> {
```

Comment:
Why `any` here? And below on line 110.

Reply:
There isn't an exposed ethers type for transacations now